app/controllers/home_controller.rb

```diff
@@ -4,11 +4,22 @@ def index
 
     redirect_to setup_instructions_path unless current_organisation.ips.present?
 
+    connections = UseCases::Administrator::GetAuthRequests.new(
+      authentication_logs_gateway: Gateways::Sessions.new(
+        ips: current_organisation.ips.map(&:address)
+      )
+    ).execute(
+      username: nil,
+      ips: @ips
```

Comment:
This variable (`@ips`) isn't defined until [line 7](https://github.com/alphagov/govwifi-admin/pull/409/files#diff-14dfef9f01460f69f175a8e6bd7d143dR16), so it'll be `nil` at this point. 

You'll need to define `@ips` before you call  `execute` on this use case.